setup.py

```diff
@@ -36,7 +36,8 @@
         'elasticsearch',
         'Jinja2==2.7.2',
         'influxdb==0.1.13',
-        'pywinrm==0.0.3'
+        'pywinrm==0.0.3',
+        'wagon'
```

Comment:
please add a comment above the `wagon` dependency that it is intentionally left with no version because it better reflects the user use case in which (s)he will always use the latest version while we are hardcoding a version in the plugin installer